strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/p2/Artifact.java

```diff
@@ -0,0 +1,46 @@
+package org.carlspring.strongbox.providers.layout.p2;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+public class Artifact
```

Comment:
Don't you think it will be redundant considering the package name? The FQN will become org.carlsping.strongbox.providers.layout.p2.P2Artifact